### ISSUE TITLE
Search block is missing styles

### ISSUE THREAD

**Issue by kjellr** [WordPress/twentytwentytwo#11]
Current state:

<img width="680" alt="Screen Shot 2021-10-05 at 1 19 12 PM" src="https://user-images.githubusercontent.com/1202812/136071809-f5d1ed67-a581-4dfd-864a-13a140635cfb.png">

**Comment by MaggieCabrera** [comment]
I had the markup for testing this block around, I'm dropping it here for easier testing:

```
<!-- wp:search {"label":"Search","placeholder":"Placeholder text...","buttonText":"Search"} /-->

<!-- wp:search {"label":"Search","showLabel":false,"placeholder":"Placeholder text...","buttonText":"Search"} /-->

<!-- wp:search {"label":"Search","placeholder":"Placeholder text...","buttonText":"Search","buttonPosition":"button-inside"} /-->

<!-- wp:search {"label":"Search","showLabel":false,"placeholder":"Placeholder text...","buttonText":"Search","buttonPosition":"button-inside"} /-->

<!-- wp:search {"label":"Search","placeholder":"Placeholder text...","buttonText":"Search","buttonUseIcon":true} /-->

<!-- wp:search {"label":"Search","showLabel":false,"placeholder":"Placeholder text...","buttonText":"Search","buttonUseIcon":true} /-->

<!-- wp:search {"label":"Search","placeholder":"Placeholder text...","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true} /-->

<!-- wp:search {"label":"Search","showLabel":false,"placeholder":"Placeholder text...","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true} /-->

<!-- wp:search {"label":"Search","placeholder":"Placeholder text...","buttonText":"Search","buttonPosition":"no-button"} /-->

<!-- wp:search {"label":"Search","showLabel":false,"placeholder":"Placeholder text...","buttonText":"Search","buttonPosition":"no-button"} /-->
```

How it looks like on TT2:

<img width="757" alt="Screenshot 2021-10-06 at 19 36 05" src="https://user-images.githubusercontent.com/3593343/136254955-23f2b43a-12f9-4654-a6fd-a0022c7754b0.png">

**Comment by carolinan** [comment]
Is the correct style for the search block the one in [the Figma template for the 404 page?](https://www.figma.com/file/76mfUcaK4QDlrXElk8MK3H/Twenty-Twenty-Two?node-id=815%3A4945)

It's been a year since I used Figma :)  -is the input field and button supposed to be 70px tall?

**Comment by kjellr** [comment]
I think we should actually just work backwards from the button: 

- The search button by default should use the same text size and padding as the normal button block. 
- Then the search field should be as tall as that button, so they line up.

**Comment by carolinan** [comment]
I believe this will need custom CSS because it can't be changed with theme.json:
https://github.com/WordPress/gutenberg/issues/34216

**Comment by kjellr** [comment]
I opened an issue upstream to track this: https://github.com/WordPress/gutenberg/issues/36444

**Comment by jffng** [comment]
I think we should hard code the styles in CSS, and remove them as soon as global styles supports styling the block.

**Comment by kjellr** [comment]
Let's give that a try and see how complicated/verbose the code needs to be.

**Comment by kjellr** [comment]
Fixed by #235.
